### ISSUE TITLE
SFTP to boxes that don't allow SSH

### ISSUE THREAD

**Issue by dpribyla** [Microsoft/vsts-tasks#4613]
I am trying to SFTP to machines that have SSH blocked.  I am able to connect to them with WinSCP.  

I was looking at the issues and found  #3195 so I tried Copy Files over SSH task.

When I use that I get the following error:

 Error: Timed out while waiting for handshake..

Is it even possible to use the Copy Files over SSH when SSH is blocked on the machines?  Is there another task I can use to just to straight   SFTP?

**Comment by DavidStaheli** [comment]
Hi @dpribyla, when you say SSH is blocked, is FTP open and you're trying to use FTPS?  SFTP is secure SSH and won't work if SSH is blocked, but FTPS is secure FTP and you can use the FTP Upload task for that.

**Comment by dpribyla** [comment]
It was my mistake.  I was running from a box that didn't have connectivity to the box I wanted to SFTP.  Once I moved it to a machine that did have connectivity I was able to SFTP successfully.  Thanks for your help.
